packages/components/src/styles/components/stepper/task-indicator.scss

```diff
@@ -7,31 +7,36 @@
 // STEPPER > INDICATOR > TASK
 //
 
-$hds-stepper-indicator-task-statuses: ( "incomplete", "progress", "processing", "complete" );
+$hds-stepper-indicator-task-statuses: (
+  "incomplete",
+  "progress",
+  "processing",
+  "complete"
+);
 $hds-stepper-indicator-task-size: 16px;
 
 // Determine states and corresponding styles
 $hds-stepper-indicator-task-status-props: (
   "incomplete": (
     "color-default": var(--token-color-palette-neutral-300),
-    "color-hover": var(--token-color-palette-blue-300),
-    "color-active": var(--token-color-palette-blue-400),
+    "color-hover": var(--token-color-foreground-action-hover),
```

Comment:
It's possible to map all of these colors to semantic variables, but in this scenario it seems logical to do it where we can.